docs/src/components/Fragment.tsx

```diff
@@ -21,7 +21,7 @@ export interface FragmentProps {
    * Note: if this is true, platforms={['web']} should be used to enable web content instead of platforms={['react', 'vue', 'angular']}
    */
   useCommonWebContent?: boolean;
-  children: ({ platform: string }) => LoaderComponent;
+  children: ({ platform }: { platform: string }) => LoaderComponent;
```

Comment:
Dumb question: why does `platform: string` not work?

Reply:
`{ platform: string }` means to rename `platform` to `string`, so the linter complains the var `string` is declared but never used 😂